packages/scripts/src/upgrade-tests/voting2/0_Deploy.ts

```diff
@@ -48,40 +36,84 @@ async function main() {
-    "FixedSlashSlashingLibrary"
-  );
   // baseSlashAmount: amount slashed for missing a vote or voting wrong.
   const baseSlashAmount = hre.ethers.utils.parseUnits("0.001", "ether");
+
   // governanceSlashAmount: amount slashed for voting wrong in a governance vote.
   const governanceSlashAmount = hre.ethers.utils.parseUnits("0", "ether");
-  const slashingLibrary = await slashingLibraryFactory.deploy(baseSlashAmount, governanceSlashAmount);
-
-  console.log("Deployed SlashingLibrary: ", slashingLibrary.address);
 
-  console.log("2. DEPLOYING VOTING V2");
-  const votingV2Factory: VotingV2Ethers__factory = await getContractFactory("VotingV2");
   const emissionRate = "0"; // Initially set the emission rate to 0.
   const unstakeCooldown = 60 * 60 * 24 * 7; // 7 days
-  const phaseLength = "86400";
-  const gat = hre.ethers.utils.parseUnits("5500000", "ether"); // Set the GAT to 5.5 million tokens.
+  const phaseLength = "86400"; // 1 day
+  const gat = hre.ethers.utils.parseUnits("5000000", "ether"); // Set the GAT to 5.0 million tokens.
 
-  // Set the SPAT to 25%. This is the percentage of staked tokens that must participate to resolve a vote.
-  const spat = hre.ethers.utils.parseUnits("0.25", "ether");
+  // Set the SPAT to 50%. This is the percentage of staked tokens that must participate to resolve a vote.
+  const spat = hre.ethers.utils.parseUnits("0.5", "ether");
 
-  // A price request can roll, at maximum, 2 times before it is auto deleted (i.e on the 3rd roll it is auto deleted).
-  const maxRolls = 2;
+  // A price request can roll, at maximum, 4 times before it is auto deleted (i.e on the 3rd roll it is auto deleted).
+  const maxRolls = 4;
 
   // The maximum number of requests that can be placed within a single round. If exceeded, the request will auto roll.
   const maxRequestsPerRound = 1000;
 
+  // ProposerV2 default bond
+  const proposerV2DefaultBond = hre.ethers.utils.parseUnits("5000", "ether");
+
+  // DVM upgrader address
+  const votingUpgraderAddress = process.env[UPGRADER_WALLET] || (await hre.ethers.getSigners())[0].address;
+
+  console.log("DVM2.0 Parameters:");
+  console.table({
+    emergencyQuorum: hre.ethers.utils.formatUnits(emergencyQuorum, "ether"),
+    emergencyExecutor,
+    baseSlashAmount: hre.ethers.utils.formatUnits(baseSlashAmount, "ether"),
+    governanceSlashAmount: hre.ethers.utils.formatUnits(governanceSlashAmount, "ether"),
+    emissionRate: hre.ethers.utils.formatUnits(emissionRate, "ether"),
+    unstakeCooldown,
+    phaseLength,
+    gat: hre.ethers.utils.formatUnits(gat, "ether"),
+    spat: hre.ethers.utils.formatUnits(spat, "ether"),
+    maxRolls,
+    maxRequestsPerRound,
+    proposerV2DefaultBond: hre.ethers.utils.formatUnits(proposerV2DefaultBond, "ether"),
+    votingUpgraderAddress,
+  });
+
+  // End DVM2.0 parameters
+
+  const readline = require("readline").createInterface({
```

Comment:
nit: any reason we can't import this at the top of the file like we do with other modules?